libs/blocks/global-navigation/delayed-utilities.js

```diff
@@ -72,7 +73,11 @@ class MenuControls {
       }
     } else {
       curtain.classList.add('is-open');
-      el.querySelector('.gnav-search-input').focus();
+      // Search template is not available yet
+      // TODO: better logic to focus elements that have not been added yet
+      if (el.querySelector('.feds-search-input')) {
```

Comment:
I'll make a note to change it in a future iteration. I honestly haven't decided on which is better, I think I prefer the one-liner you've referenced to avoid too many `if` clauses.